### PR TITLE
Core monitor exception list

### LINKED ISSUE #352: New-CmAzMonitorLogAlerts - Exclusion List
We frequently monitor machines or services which should be excluded from standard alerts. A good example is cache disks which are almost always full, and therefore always fire alerts.

To cut down on noise we want to add an exclusion list. This should exist within take the format:

**CmLogExclusionList_CL**
| AlertName | ResourceId |
|-------------|--------------|
| High_CPU_Usage_Percent | `fredev01` |
| High_CPU_Usage_Percent_Range | `fredev01` |
| Low_Disk_Space | `fretstdc001` |

We should then update the YAML document:

``` yaml
groups:
- name: VMStandard
  alertSets:
  - name:                                                                                             # Optional - It is recommended to use name when using multiple sets of alerts.
    type: VirtualMachines
    alerts:
    - name: Low-Dsk-Crt
      definition: Low_Disk_Space
      description: Critically low available disk space percentage and MB value.
      severity: Critical
      parameters:
        _maxPercentage: 5
        _maxValueMB: 102400
        _timeWindow: 5m
      exclusions:      # <-- This is the new section
        - fredev01
        - fretstdc001
```

We should loop through these exclusions and form the `CmExclusionList_CL` list table. We should then use [code like this](https://www.powershellgallery.com/packages/Upload-AzMonitorLog/1.2/Content/Upload-AzMonitorLog.ps1) to upload an exclusion list, and modify `standardDefinitions.yml` queries to join on the exclusion list and resource ID to ensure alerts ignore the exclusions.

### PR DESCRIPTION
**Pull request by deepend-dev**
closes #352

### CODE REVIEW

**Review by nathankitchen — CHANGES_REQUESTED** [review]
Do we not have a schema for validating the `log.yml`? Either:
 - [ ] There isn't a log schema and we should add one with a unit test
Or
 - [x] There is a log schema and we need to add `exclusions` to it, and also add a unit test to check it
Other than that, it looks good.

File: Cloudmarque.Azure/Resources/Project/Integration/alerts/log.yml [hunk 1]
```diff
@@ -18,10 +18,15 @@ groups:
   - name:                                                                                             # Optional - It is recommended to use name when using multiple sets of alerts.
     type: VirtualMachines
     alerts:
-    - name: High-CPU-Crt                                                                              # Optional - It is recommended to use name for more understandable alert names.
+    - name: High-CPU-Deep-Test-Crt                                                                              # Optional - It is recommended to use name for more understandable alert names.
```
[the remark this answers is not in the document]
Comment (deepend-dev) [reply Cloudmarque.Azure/Resources/Project/Integration/alerts/log.yml:21]: missed it. sorry. will correct all changes

File: Cloudmarque.Azure/Public/Deploy/Monitor/logAlerts/standardDefinitions.yml [hunk 2]
```diff
@@ -47,11 +50,14 @@ Queries:
       _minPercentage: 95
       _maxPercentage: 99
       _timeWindow: 30m
+    propertiesAvailble:
```
[the remark this answers is not in the document]
Comment (deepend-dev) [reply Cloudmarque.Azure/Public/Deploy/Monitor/logAlerts/standardDefinitions.yml]: done

File: Cloudmarque.Azure/Resources/Project/Integration/alerts/log.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (deepend-dev) [reply Cloudmarque.Azure/Resources/Project/Integration/alerts/log.yml]: done

**Comment by deepend-dev** [comment]
Updated scripts as per review

**Pull request merged**